src/transactions/simulation/TxSimUtils.cpp

```diff
@@ -387,6 +387,7 @@ generateScaledLiveEntries(
 
             break;
         }
+        case SPEEDEX_CONFIGURATION:
```

Comment:
I might be missing something, but I don't think this is right. I think it will crash the simulation.

Reply:
I'm not sure how to test it (apparently it's not covered in tests?), changed to just break for now.